tests/TestRun2/UserThreadTest.jmx

```diff
@@ -71,17 +71,17 @@
           </elementProp>
           <elementProp name="stockFile" elementType="Argument">
             <stringProp name="Argument.name">stockFile</stringProp>
-            <stringProp name="Argument.value">tests/TestRun2/ConfigstockIds.csv</stringProp>
+            <stringProp name="Argument.value">\TestRun2\Config\stockIds.csv</stringProp>
```

Comment:
No need to commit this file

Reply:
I convert it back. And just ignore file for everyone next push. This can be merged fine